Markdown_to_TeX/Compile.bat

```diff
@@ -1,12 +1,17 @@
 @echo OFF
```

Comment:
an equivalent shell script should be created in order to run this in non-windows environments (for example, I use `.sh` scripts with Windows Subsystem for Linux and almost never touch Powershell)

Reply:
Want to do this in a separate cross-platform branch